README.md

```diff
@@ -8,63 +8,39 @@
+pgvecto.rs is a Postgres extension that provides vector similarity search functions. It is written in Rust and based on [pgrx](https://github.com/tcdi/pgrx). It is currently ⚠️**under heavy development**⚠️, please take care when using it in production.
 
 ## Why use pgvecto.rs
 
 - 💃 **Easy to use**: pgvecto.rs is a Postgres extension, which means that you can use it directly within your existing database. This makes it easy to integrate into your existing workflows and applications.
 - 🦀 **Rewrite in Rust**: Rewriting in Rust offers benefits such as improved memory safety, better performance, and reduced **maintenance costs** over time.
 - 🙋 **Community**: People loves Rust We are happy to help you with any questions you may have. You could join our [Discord](https://discord.gg/KqswhpVgdU) to get in touch with us.
 
-## Why not a specialty vector database?
-
-Imagine this, your existing data is stored in a Postgres database, and you want to use a vector database to do some vector similarity search. You have to move your data from Postgres to the vector database, and you have to maintain two databases at the same time. This is not a good idea.
-
-Why not just use Postgres to do the vector similarity search? This is the reason why we build pgvecto.rs. The user journey is like this:
+## Installation from Source
 
-```sql
--- Update the embedding column for the documents table
-UPDATE documents SET embedding = ai_embedding_vector(content) WHERE length(embedding) = 0;
+We're working on binary release with deb package. Currently, you need to build from source.
 
--- Create an index on the embedding column
-CREATE INDEX ON documents USING vectors (embedding l2_ops) WITH (algorithm = "HNSW");
-
--- Query the similar embeddings
-SELECT * FROM documents ORDER BY embedding <-> ai_embedding_vector('hello world') LIMIT 5;
-```
-
-From [SingleStore DB Blog](https://www.singlestore.com/blog/why-your-vector-database-should-not-be-a-vector-database/):
-
-> Vectors and vector search are a data type and query processing approach, not a foundation for a new way of processing data. Using a specialty vector database (SVDB) will lead to the usual problems we see (and solve) again and again with our customers who use multiple specialty systems: redundant data, excessive data movement, lack of agreement on data values among distributed components, extra labor expense for specialized skills, extra licensing costs, limited query language power, programmability and extensibility, limited tool integration, and poor data integrity and availability compared with a true DBMS.
-
-## Setting up the development environment
-
-You could use [envd](https://github.com/tensorchord/envd) to set up the development environment with one command. It will create a docker container and install all the dependencies for you.
+### Install Rust and base dependency
 
 ```sh
-pip install envd
-envd up
+apt install -y build-essential libpq-dev libssl-dev pkg-config gcc zlib1g-dev
```

Comment:
libreadline-dev flex bison libxml2-dev libxslt-dev libxml2-utils xsltproc ccache clang

Reply:
Added without ccache and clang, hence I added gcc already

Reply:
@VoVAllen Crate `bindgen` needs `clang-sys`. It's necessary. I'm not sure whether `ccache` is needed. The [link](https://wiki.postgresql.org/wiki/Compile_and_Install_from_source_code) says yes.

Reply:
added